gplugins/tidy3d/component.py

```diff
@@ -663,23 +664,20 @@ def write_sparameters_batch(
         cross_section=cross_section,
     )  # Coupler Strip C-Band
 
-    sps = write_sparameters_batch(
+    sims = write_sparameters_batch(
         [
             dict(
                 component=coupler_sc(length=i),
                 # sim_size_z=0,
-                filepath=PATH.sparameters_repo / f"dc_{i}.npz",
+                # filepath=PATH.sparameters_repo / f"dc_{i}.npz",
                 layer_stack=layer_stack,
-                # overwrite=True,
+                sim_size_z=0,
+                overwrite=True,
```

Comment:
Do we want `overwrite=True` by default here?

Reply:
this would take a lot of credits

Reply:
this is only on the __main__ file but i can remove it

Reply:
sure, it's fine, i was juts wondering